esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -210,15 +210,17 @@ def test_offer_claim_invalid_id(self):
         self.assertIn('Offer with name or uuid %s not found.' % fake_name,
                       e.stderr.decode())
 
-    @decorators.attr(type=['negative'])
-    def test_offer_create_invalid_node(self):
+    @pytest.mark.negative
+    def test_create_invalid_node(self):
         """ Tests that "esi [offer/lease] create" properly handles being
                 passed a resource node uuid that does not exist.
             Test steps:
             1) Attempt to create an offer for a node that does not exist.
             2) Check that the command failed. (returned non-zero exit code)
-            3) Attempt to create a lease for a node that does not exist.
-            4) Check that the command failed. (returned non-zero exit code) """
+            3) Check that the proper error message was sent to stderr.
+            4) Attempt to create a lease for a node that does not exist.
+            5) Check that the command failed. (returned non-zero exit code)
+            6) Check that the proper error message was sent to stderr. """
```

Comment:
I kind of prefer splitting this test up into two: one for offers and one for leases. Generally speaking the more focused the tests the better, as it helps immediately isolate issues due to a failing test.

Reply:
If I'm splitting these up (which is a solid call), should I split all the other tests I've written into more focused tests?

Reply:
I'd say yes!